test/e2e/ipstest/ips_get_test.go

```diff
@@ -41,7 +41,7 @@ func TestCli_Ips_Get(t *testing.T) {
 					t.Skip("Skipping this test because someCondition is true")
 				}
 				root := c.Root()
-				projectName := "metal-cli-ips-get" + helper.GenerateRandomString(5)
+				projectName := "metal-cli-ips-get-" + helper.GenerateRandomString(5)
```

Comment:
Could you move these changes to a separate PR?

Reply:
Yeah sure. Removed from this one